examples/mobile/HomeApp/index.html

```diff
@@ -42,9 +42,51 @@
 					!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='https://weatherwidget.io/js/widget.min.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','weatherwidget-io-js');
 				</script>
 			</div>
-			<div class="ui-content-area">
-				News Widget
+			<div class="ui-content-subheader">
+				Latest News
 			</div>
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-expandable ui-li-divider">
+					<h2>
+						World
+					</h2>
+					<ul class="ui-listview">
+						<li>
+							Item 1
+						</li>
+						<li>
+							Item 2
+						</li>
+					</ul>
+				</li>
+				<li class="ui-expandable ui-li-divider">
+					<h2>
+						Technology
+					</h2>
+					<ul class="ui-listview">
+						<li>
+							Item 1
+						</li>
+						<li>
+							Item 2
+						</li>
+					</ul>
+				</li>
+				<li class="ui-expandable ui-li-divider">
```

Comment:
Can remove the ui-li-divider

Reply:
In guides there is divider between items so I would like to leave it
(see p. 92. 3-6 Expand Button of GUI Guide v2.2

Reply:
Is the ui-li-divider drawing the underline of the list element?
![image](https://user-images.githubusercontent.com/29908741/86356224-d60d2480-bca6-11ea-9d8f-83b86db7adef.png)